satpy/readers/aapp_l1b.py

```diff
@@ -54,7 +55,7 @@
 
 
 def create_xarray(arr):
-    res = da.from_array(arr, chunks=(1000, 1000))
+    res = da.from_array(arr, chunks=(CHUNK_SIZE, CHUNK_SIZE))
     res = xr.DataArray(res, dims=['y', 'x'],
                        coords=[np.arange(res.shape[0]),
                                np.arange(res.shape[1])])
```

Comment:
Doesn't xarray create these coordinates on the fly if needed?

Reply:
no unfortunately

Reply:
The coordinates could be `da.arange` though, right? Plus since these are just aranges I don't think anything would use them. If you did operations with a non-coords array that had y/x dimensions and one that did I think it fills in the blanks.